bitnami/solr/values.yaml

```diff
@@ -0,0 +1,655 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+#   storageClass: myStorageClass
+
+## String to partially override kafka.fullname template (will maintain the release name)
+##
+# nameOverride:
+
```

Comment:
Ran a check, these elements defined in README.md are not defined here (NOTE: there may be false negatives because of the commented values). Please double check

```
Element cloudBootsrap in README not found in values.yaml
Element containerSecurityContext.sysctls in README not found in values.yaml
Element existingSecret in README not found in values.yaml
Element exporter.extraArgs in README not found in values.yaml
Element exporter.image.pullSecrets in README not found in values.yaml
Element exporter.service.loadBalancerIP in README not found in values.yaml
Element fullnameOverride in README not found in values.yaml
Element global.imagePullSecrets in README not found in values.yaml
Element global.imageRegistry in README not found in values.yaml
Element global.storageClass in README not found in values.yaml
Element nameOverride in README not found in values.yaml
Element persistence.matchExpressions in README not found in values.yaml
Element persistence.matchLabels in README not found in values.yaml
Element persistence.path in README not found in values.yaml
Element persistence.storageClass in README not found in values.yaml
Element persistence.subPath in README not found in values.yaml
Element podDisruptionBudget in README not found in values.yaml
Element podSecurityContext.sysctls in README not found in values.yaml
Element podSecurityPolicy.create in README not found in values.yaml
Element service.loadBalancerIP in README not found in values.yaml
```

Reply:
Forget about cloudBootstrap, it's a typo in the README (cloudBootsrap)